server/utils/manualJobNotification.js

```diff
@@ -0,0 +1,69 @@
+require('dotenv').config();
```

Comment:
I think all these logic could happen in a utility function and passed to the generic notify() function